pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -851,8 +851,15 @@ func GetContentFromFile(file string) (string, error) {
 // FormatEnvName format env name
 func FormatEnvName(name string) string {
 	envName := strings.Trim(name, "./")
+	// only take string after the last slash only if the string contains a slash
+	if strings.Contains(envName, "/") {
```

Comment:
Thank you! This helps, does this break anything though @AhmedGrati with regards to how we use env?

This also cleans up the really long configmap names, but I'm unsure if they would have conflicts when there are multiple containers.

If you can also add a test case for this under k8sutils_test.go that'd be awesome. Otherwise, good PR 👍

Reply:
Thanks!
I think it will work as long as env_file names are unique. When different env_file's with the same name are references (from different directories) it will clash. 
Could also just only cap at the last, let's say 60 chars? And not filter by last `/` ?

Reply:
Do you want any changes? If not could it be merged soon? I could really use this in a new version of Kompose.

Reply:
You're right, I think that the names generated should be from env_file names from the last slash.. I really doubt that there will be multiple env file names in different directories for a docker compose project.

HOWEVER if there is, I think that they should be the same name, but one of them `filename_1`, etc. if there are more of the same file AND Kompose does a warning.

However, I don't feel comfortable merging the PR until we have these two tasks:
- [ ] tests added to k8sutils_test.go
- [ ] test for if there are two env file names in different directories
- [ ] implement renaming to filename_number if there are multiple env files of the same name but in different directories.

Reply:
Ok, just took a short browse. Added a new `env-multiple` fixture and test but now sure how to implement this. My thinking would be to check existing configmaps before inserting the new. If a configmap with the same name exists, append a 1 and try again. If that also exists a 2, and so on..
Could you give me a pointer? Or perhaps someone else wants to do this.

Reply:
Sorry, I didn't actually get it @ThijsBroersen. Can you please clarify it more?